pkgs/build-support/dotnet/build-dotnet-module/default.nix

```diff
@@ -188,18 +190,18 @@ stdenvNoCC.mkDerivation (args // {
             esac
         done
 
+        export tmp=$(mktemp -td "${pname}-tmp-XXXXXX")
+        HOME=$tmp/home
+
         exitTrap() {
             test -n "''${ranTrap-}" && return
             ranTrap=1
 
             if test -n "''${keepSources-}"; then
-                echo -e "Path to the source: $src\nPath to the fake home: $HOME"
+                echo -e "Path to the source: $tmp/src\nPath to the fake home: $tmp/home"
             else
-                rm -rf "$src" "$HOME"
+                rm -rf "$tmp"
             fi
-
-            # Since mktemp is used this will be empty if the script didnt succesfully complete
-            ! test -s "$depsFile" && rm -rf "$depsFile"
```

Comment:
Why was this removed?

Reply:
That was a mistake, at one point I had moved the file into $tmp, but put it back and forgot to undo this change.